### ISSUE TITLE
✨ [Feature] Add a orm adapter for casbin

### ISSUE THREAD

**Issue by devhaozi** [goravel/goravel#256]
### Before feedback (在反馈之前)

- [X] There are no features that I want to submit in Issues (当前 Issues 中没有我将要提交的新功能)

### Describe Feature (描述功能)

Currently we can use `facades.Orm().DB()` to get a `sql.DB` and use Casbin with `github.com/Blank-Xu/sql-adapter`, but I think it is best to develop a special adapter for Goravel.

**Comment by devhaozi** [comment]
Casbin doc: [https://casbin.org/zh/docs/adapters/](https://casbin.org/zh/docs/adapters/)

**Comment by devhaozi** [comment]
Or we can try to develop an package, just like `github.com/spatie/laravel-permission`. It is still troublesome to use Casbin in Goravel now.

**Comment by hwbrzzl** [comment]
Yes, it's a good idea to create a new package to deal with this feature.

**Comment by dnslin** [comment]
@hwbrzzl how Goravel now thinks that Casbin can handle RBAC-like permissions management as conveniently

**Comment by hwbrzzl** [comment]
Sorry, we haven't thought this deeply yet. But there is a third package: https://github.com/wcz0/goravel-authz, you can check it first if needed.

**Comment by devhaozi** [comment]
https://github.com/wcz0/goravel-authz
